docs/tutorial/getting_started.md

```diff
@@ -0,0 +1,77 @@
+# Tutorial
+
+```eval_rst
+.. contents::
+```
+
+This guide is meant to give you a simple overview of how to use opsdroid, and to kickstart you projects. It is in no way complete, and I highly recommend checking out the docs for more advanced usage, and a more in-depth overview.
+
+## Getting Started
+
+### What is Opsdroid
+
+Opsdroid is a framework to make creating and extending your ChatOps workflows powerful but simple. Opsdroid is a modular framework. It's event loop is power by the `asyncio` framework. The event loop starts with connectors, which allow Opsdroid to interface with services like Slack and Telegram, then Opsdroid sends the event to parsers and matchers, which will determine which skill to run.
```

Comment:
This could be added/merged into the first paragraph on the index page, I don't think it belongs here.

https://github.com/opsdroid/opsdroid/blob/master/docs/index.md

Reply:
Okay, good idea. Though the idea of this is sort of having a one-stop-shop, so maybe we keep it both places?

Reply:
Happy to have something here, but it should be short and snappy. Maybe one sentence or two.

Perhaps focus on what it's for rather than how it works.

Reply:
Sure thing!

Reply:
It's basically just a leaner version of the overview, so I will get rid of it.